### ISSUE TITLE
os.read_lines function adds additional control characters to line strings

### ISSUE THREAD

**Issue by progamateur** [vlang/v#3697]
**V version:** V 0.1.25 4bb5d7d
**OS:** windows 10 (1909)

**What did you do?**
Tried to use os.read_lines() function to get the line strings of a text file. See the attached code that illustrates the issue.
 

[test_os_read_lines.zip](https://github.com/vlang/v/files/4175625/test_os_read_lines.zip)

**What did you expect to see?**

Just the string content of the lines.
 
**What did you see instead?**

Extra control character(s) at the end of the line strings : 
  \r  at the end of all line strings,
  \r\n at the end of the last line string

**Comment by alexesprit** [comment]
<details>
  <summary>Example from description (Click to expand)</summary>

```v
 import ( os )
 fn main() {
	file_name := 'test_file.txt' 	
	os.system('echo line_1>${file_name}')
	os.system('echo line_2>>${file_name}')
	os.system('echo line_3>>${file_name}')
	os.system('echo line_4>>${file_name}')
	
	println('// Printing of "test_line.txt" file using "type" cmd')
	os.system('type ${file_name}')
	
	mut lines := os.read_lines('test_file.txt') or {
		println('Test_file.txt reading issue !') return
	}
	
	println('\n// Printing of lines string without processing')
	for line in lines {
		println('*${line}+')
	}

	println('\n// Printing of lines string after right-triming "\\r" all lines string')
	for line in lines {
		modified_line := line.trim_right('\r')
		println('*${modified_line}+')
	}

	println('\n// Printing of lines string after right-triming "\\r" all lines string except the last')
	println('//                            and right-triming "\\r\\n" the last line string')
	lines[lines.len - 1] = lines[lines.len - 1].trim_right('\n')
	for line in lines {
		modified_line := line.trim_right('\r')
		println('*${modified_line}+')
	}
}
```
</details>

**Comment by alexesprit** [comment]
Currently, strings are splitted into lines with `\n` character only, and it fails if a text file is created with `\r\n` characters as a line separator.

**Comment by alexesprit** [comment]
Fixed in #3699.

```v
import os

fn main() {
	print_text('test_file_lf.txt')
	print_text('test_file_crlf.txt')
}

fn print_text(path string) {
	lines := os.read_lines(path) or {
		println('Unable to read $path')
		return
	}

	println('Print lines from $path:')
	for line in lines {
		println('*${line}+')
	}
}
```
```
Print lines from test_file_lf.txt:
*line_1+
*line_2+
*line_3+
*line_4+
Print lines from test_file_crlf.txt:
*line_1+
*line_2+
*line_3+
*line_4+
```

**Comment by progamateur** [comment]
Hello alexesprit, medvednikov 
Using V 0.1.25 3f5e4c5,
os.read_lines function processes correctly my text files now (without preproccesing the line strings).
Thanks
